pyproject.toml

```
@@ -69,6 +69,10 @@ types-pyyaml = "^6.0.12.10"
 types-python-slugify = "^8.0.0.2"
 pre-commit = "^3.3.3"
 jsonschema = "^4.22.0"
+semver = "^3.0.2"
+pillow = "^10.3.0"
+numpy = "^1.26.4"
+pyastgrep = "^1.3.2"
```

I suggest moving these dependencies (with `jsonschema`) to a dedicated group. They are not related to test but to the cli.

```suggestion

[tool.poetry.group.scripts.dependencies]
jsonschema = "^4.22.0"
semver = "^3.0.2"
pillow = "^10.3.0"
numpy = "^1.26.4"
pyastgrep = "^1.3.2"
```